vendor/bat-native-ads/src/bat/ads/internal/database/tables/conversions_database_table.cc

```diff
@@ -259,6 +264,46 @@ void Conversions::MigrateToV10(mojom::DBTransaction* transaction) {
   util::CreateIndex(transaction, "creative_ad_conversions", "creative_set_id");
 }
 
+void Conversions::MigrateToV20(mojom::DBTransaction* transaction) {
+  DCHECK(transaction);
+
+  util::Rename(transaction, "creative_ad_conversions",
+               "creative_ad_conversions_temp");
+
+  const std::string& query = base::StringPrintf(
+      "CREATE TABLE ad_conversions "
```

Comment:
should it be `creative_ad_conversions` table ?

Reply:
Great spot,thanks

Reply:
Fixed